sig-etcd/charter.md

```diff
@@ -0,0 +1,63 @@
+# SIG etcd Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+
+## Scope
+
+Owns the etcd project and how it is used by Kubernetes.
```

Comment:
note that you gain ownership of the current, very cumbersome and undeterministic process of updating etcd server/client in k8s.

this is currently a best effort from community members and issues and PRs just run stale:
https://github.com/kubernetes/kubernetes/issues/117648
https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+label%3Aarea%2Fkubeadm+etcd

various product tooling allows

Reply:
Historically it were etcd maintainers who bumped the etcd image in Kubernetes. Reason was that Kubernetes scalability tests are the major signal in etcd qualification, so minor bumps etcd version were done immediately. 

What's more cumbersome is security patching, which is a problem because there are the etcd k8s image is also used by kubeadm. With the SIG in place, I think we can discuss changing/improving the release process.

Reply:
> Historically it were etcd maintainers who bumped the etcd image in Kubernetes. Reason was that Kubernetes scalability tests are the major signal in etcd qualification, so minor bumps etcd version were done immediately.

i think i vaguelly recall this period.

> What's more cumbersome is security patching, which is a problem because there are the etcd k8s image is also used by kubeadm. With the SIG in place,

the biggest current pain points for etcd update in k8s are:
- undocumented, manual image promotion process.
- no dedicated top level approver to own etcd k/k updates as it touches a lot of things.
- too many steps, dufferent repos, client vs server updates

kubeadm updates are simple - changing a small version map / constant. by updating kubeadm k8s+etcd gains upgrade signal from etcd version at k8s N-1 to etcd version at k8s N. the kube-up upgrade suite is still not working, IIRC.

>  I think we can discuss changing/improving the release process.

happy to participate.